code.gs

```diff
@@ -1169,8 +1169,12 @@ function htmlEscape (str) {
 function isRunningOutdatedVersion () {
   var response, latestVersion;
 
-  response = UrlFetchApp.fetch(baseGitHubApiURL + 'releases/latest');
-  if (response.getResponseCode() !== 200) {
+  try {
+    response = UrlFetchApp.fetch(baseGitHubApiURL + 'releases/latest');
+    if (response.getResponseCode() !== 200) {
+      throw new Error('');
+    }
+  } catch (err) {
     log.add('Unable to get the latest version number: the requested URL returned a ' + response.getResponseCode() + ' response.', 'warning');
```

Comment:
If you will add both the fetch and the response-code inside the same try-block, in the catch-block (the log.add) you need to handle the case where the initial fetch failed, and there is no response.getResponseCode() method to call. For example something like (not tested):
```javascript
  try {
    response = UrlFetchApp.fetch(baseGitHubApiURL + 'releases/latest');
    if (response.getResponseCode() !== 200) {
      throw new Error('');
    }
  } catch (err) {
    log.add('Unable to get the latest version number' + (response.getResponseCode ? ': the requested URL returned a ' + response.getResponseCode() + ' response.' : ''), 'warning');
```